menas/src/main/scala/za/co/absa/enceladus/menas/services/DatasetService.scala

```diff
@@ -217,6 +215,15 @@ class DatasetService @Autowired()(datasetMongoRepository: DatasetMongoRepository
     }
   }
 
+  def getLatestVersionsWithMissingProperty(missingProperty: Option[String]): Future[Seq[Dataset]] = {
+    val missingFilter = missingProperty match {
+      case Some(missingProp) => Filters.not(Filters.exists(s"properties.${missingProp}"))
```

Comment:
Yes, I was thinking about this too, but it should be the number of datasets missing for each property. If you have `Filters.and` for all of them it would give the number of datasets missing all of these properties(or datasets which don't have any of these properties)